src/interpreter.rs

```diff
@@ -2,6 +2,35 @@ use rustpython_vm::{Interpreter, Settings, VirtualMachine};
 
 pub type InitHook = Box<dyn FnOnce(&mut VirtualMachine)>;
 
+/// The convenient way to create `rustpython_vm::Interpreter` with stdlib and other stuffs.
+///
+/// Basic usage:
+/// ```
+/// let interpreter = rustpython::InterpreterConfig::new()
+///     .init_stdlib()
+///     .interpreter();
+/// ```
+///
+/// Overriding Settings:
+/// ```
+/// let settings = Settings::Default();  // override your settings here
+/// let interpreter = rustpython::InterpreterConfig::new()
+///     .settings(settings)
+///     .interpreter();
+/// ```
+///
+/// Add native modules:
```

Comment:
grammar nitpick

Option 1: (preferred because of [parallelism](https://en.wikipedia.org/wiki/Parallelism_\(grammar\)))
```suggestion
/// Adding native modules:
```

Option 2:
```suggestion
/// To add native modules:
```